server/tso/global_allocator.go

```diff
@@ -204,94 +303,95 @@ func (gta *GlobalTSOAllocator) SyncMaxTS(ctx context.Context, dcLocationMap map[
 			}
 		}
 		// Prepare to make RPC requests concurrently
-		respCh := make(chan *pdpb.SyncMaxTSResponse, len(leaderURLs))
-		errCh := make(chan error, len(leaderURLs))
-		var errList []error
+		respCh := make(chan *syncResp, len(leaderURLs))
 		wg := sync.WaitGroup{}
+		request := &pdpb.SyncMaxTSRequest{
+			Header: &pdpb.RequestHeader{
+				SenderId: gta.allocatorManager.member.ID(),
+			},
+			SkipCheck: skipCheck,
```

Comment:
`SkipCheck` is confused for me, how about `Phase: Collect` or others(prewrite)?

Reply:
The term `SkipCheck` is a flag that indicates whether we should do the TSO check before writing it. If it's true, we could write the `maxTSO` directly without any consideration. If not, we need to make a check before writing to make sure it's the biggest one.

Reply:
Maybe it's a quite clear name here if I write some comments to explain?